package.json

```diff
@@ -14,7 +14,8 @@
     "build": "tsc -p tsconfig.build.json",
     "lint": "eslint --color --ext .ts src/ test/",
     "lint:fix": "yarn run lint --fix",
-    "test": "mocha"
+    "test": "mocha",
+    "test:dev": "mocha --timeout 36000000"
```

Comment:
do we even need this command?

Reply:
There is no need but it is pretty helpful for debugging metamask UI flow